src/components/molecules/CakeType.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import Paragraph from '../atoms/Paragraph';
+import theme from '../../utils/theme/theme';
+import { mobileQuery, tabletQuery, desktopQuery, laptopQuery } from '../../utils/mediaqueries';
+
+const TypeWrapper = styled.div({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'flex-end',
+  height: '265px',
+  [mobileQuery]: {
+    margin: '0 20px 30px',
+  },
+
+  [tabletQuery]: {
+    margin: '0 20px',
+  },
+
+  [laptopQuery]: {
+    margin: '0 40px',
+  },
+
+  [desktopQuery]: {
+    margin: '10px 50px',
+  },
+});
+
+const ListItem = styled.li(
+  {
+    fontFamily: theme.fontFamilies.dinPro.regular,
+    fontSize: theme.fontSizes.medium,
+    color: theme.colors.mediumGold,
+  },
+  props => ({
+    marginTop: props.marginTop,
+  }),
+);
+
+const UnorderList = styled.ul({
+  listStyleType: 'none',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'flex-end',
+  paddingInlineStart: '0px',
```

Comment:
what does this do? is it needed?

Reply:
Its needed because if the paddingInlineStart has some different value there is an indentation of the items